docs/sql-ref-syntax-qry-select-groupby.md

```diff
@@ -88,6 +89,41 @@ aggregate_name ( [ DISTINCT ] expression [ , ... ] ) [ FILTER ( WHERE boolean_ex
      (product, warehouse, location), (warehouse), (product), (warehouse, product), ())`.
     The N elements of a `CUBE` specification results in 2^N `GROUPING SETS`.
 
+* **Partial Grouping Analytics**
+
+    Partial grouping analytics means there are both `group_expression` and `CUBE|ROLLUP|GROUPING SETS`
+    in GROUP BY clause. For example:
```

Comment:
Can we describe the behavior with documents instead of examples? We can also take a look at how other databases document this feature.

Reply:
> Can we describe the behavior with documents instead of examples? We can also take a look at how other databases document this feature.

Oracle document https://docs.oracle.com/cd/E11882_01/server.112/e25554/aggreg.htm#DWHSG8612
![image](https://user-images.githubusercontent.com/46485123/113286635-10335180-931f-11eb-874d-78d6caecf4c2.png)
![image](https://user-images.githubusercontent.com/46485123/113286658-14f80580-931f-11eb-8ae8-0fff412170cc.png)
All have example, easier to understand.

Reply:
Oracle has descriptions as well as examples: `In this case, the GROUP BY clause creates subtotals at (2+1=3) aggregation levels. That is, at level (expr1, expr2, expr3), (expr1, expr2), and (expr1).`

Reply:
We can say that: `group_expression`s will be prepended to each group in the GROUPING SETS.